requirements.txt

```diff
@@ -1,7 +1,26 @@
 asgiref==3.2.7
+blinker==1.4
+Click==7.0
 Django==3.0.5
-django-cors-headers==3.2.1
-djangorestframework==3.11.0
-Pillow==7.1.1
+fanstatic==1.1
+Flask==1.1.1
+Flask-Mail==0.9.1
+Flask-SQLAlchemy==2.4.1
+itsdangerous==1.1.0
+Jinja2==2.11.1
+joblib==0.14.1
+js==1.0
+MarkupSafe==1.1.1
+mysqlclient==1.4.6
+numpy==1.17.4
+pygame==1.9.6
+python-decouple==3.3
 pytz==2019.3
+scikit-learn==0.22
+scipy==1.3.3
+shutilwhich==1.1.0
+sklearn==0.0
+SQLAlchemy==1.3.13
 sqlparse==0.3.1
+WebOb==1.8.6
+Werkzeug==1.0.0
```

Comment:
@akhil-maker please dont delete exisiting dependencies and dont add unnecessary dependencies. This commit needs only python-decouple to be added

Reply:
Yes sir